packages/dom/src/utils.js

```diff
@@ -8,7 +8,7 @@ export function resourceFromDataURL(uid, dataURL) {
   // build a URL for the serialized asset
   let [, ext] = mimetype.split('/');
   let path = `/__serialized__/${uid}.${ext}`;
-  let url = new URL(path, document.URL).toString();
+  let url = document.URL.includes('localhost') ? new URL(path, 'http://render.percy.local').toString() : new URL(path, document.URL).toString();
```

Comment:
lets move this logic to a function say `rewriteLocalhostURL` which would be consumed by `resourceFromDataURL` and `resourceFromText` function.

Reply:
This way we'll only add tests for that function once. 

and where it is consumed we'll have a expectation saying - `functionName.calledTimes(1)` ( similar syntax in jasmine, you get the idea)